src/components/Header/Header.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
-// import toody from '../../images/toody.svg';
-// ^^ this is the Toody logo; uncomment he above line to use it
+import toody from '../../images/toody.svg';
 import './Header.css';
 
 const Header = () => {
   return (
     <div className="header">
-      {/*  TODO:
-        You will need to complete this component. The header is the
-        "navbar" that spans the top of the page and contains the 'Toody" logo
-        at the top left and the "Profile" link at the top right. The logo
-        has already been imported for you (see line 2). The completed header
-        should route the user to the Profile page correctly. */}
+      <div id="toody-logo">
+        <img src={toody} />
+        <p id="toody-logo-text">Toody</p>
+      </div>
+
+      <div id="my-profile">
```

Comment:
it's generally best practice to not use `id` to select and style elements (more can be found via this [article](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4)). let's convert this to use `className`.